### ISSUE TITLE
Failed to setup endpoints: no session ID field

### ISSUE THREAD

**Issue by phuzybuny** [maximkulkin/esp32-homekit-camera#100]
Previews are working but live video is not. The issue does not appear to be watchdog related.

- ESP-IDF v4.4
- Ai-Thinker ESP32-CAM
- Commit: 7cb31216d160c58ff0d59944b42f59324d7998db

```
>>> HomeKit: [Client 1] Updating characteristic 1.13 ("Setup Endpoints") with TLV:
E (34245) esp32_camera: Failed to setup endpoints: no session ID field
```

Looks like `v` is NULL because `request->head->type` is 20. 
```
v = tlv_get_value(request, 1);
```

Removing the validation just results in the session ID being an invalid size.

```
E (21525) esp32_camera: Failed to setup endpoints: session ID field has invalid size (1073626880)
```

_Originally posted by @phuzybuny in https://github.com/maximkulkin/esp32-homekit-camera/issues/94#issuecomment-1138691192_

**Comment by leonrrafael** [comment]
Similar issue here. Any advice?

**Comment by daveholiday** [comment]
I'm also having this problem.  Has anyone made progress?

**Comment by leonrrafael** [comment]
No luck yet.

**Comment by rcarmo** [comment]
No luck here either.

**Comment by CP02A** [comment]
I have the same problem - has anyone succeeded at fixing the problem?

**Comment by leonrrafael** [comment]
Nope. I think the developer left the project.Em 5 de set. de 2022, à(s) 10:46, CP02A ***@***.***> escreveu:﻿
I have the same problem - has anyone succeeded at fixing the problem?

**Comment by Hellangel1987** [comment]
im now also facing this issue

![image](https://user-images.githubusercontent.com/20953591/217831356-93233941-a091-44da-8f5e-1232a27851f8.png)

E (26190) esp32_camera: Failed to setup endpoints: no session ID field

**Comment by than-sad** [comment]
Me too...
